autonode_service.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
```

Comment:
i think in my scripts i do more stringent, `set -eio`, but there are more things to turn on to make this script more robust , google please